src/Domain.LinnApps/Requisitions/RequisitionManager.cs

```diff
@@ -261,55 +264,56 @@ public async Task AddRequisitionLine(RequisitionHeader header, LineCandidate toA
+                    if (moveOnto.ToPallet.HasValue)
                     {
                         var canPutPartOnPallet = await this.requisitionStoredProcedures.CanPutPartOnPallet(
                                                      toAdd.PartNumber,
-                                                     moveOnto.Pallet.Value);
-
+                                                     moveOnto.ToPallet.Value);
                         if (!canPutPartOnPallet)
                         {
                             throw new CannotPutPartOnPalletException(
-                                $"Cannot put part {toAdd.PartNumber} onto P{moveOnto.Pallet}");
+                                $"Cannot put part {toAdd.PartNumber} onto P{moveOnto.ToPallet}");
                         }
                     }
 
                     int? locationId = null;
 
-                    if (!string.IsNullOrEmpty(moveOnto.Location))
+                    if (!string.IsNullOrEmpty(moveOnto.ToLocation))
                     {
-                        var toLocation = await this.storageLocationRepository.FindByAsync(x => x.LocationCode == moveOnto.Location);
+                        var toLocation = await this.storageLocationRepository.FindByAsync(x => x.LocationCode == moveOnto.ToLocation);
                         if (toLocation == null)
                         {
-                            throw new InsertReqOntosException($"Did not recognise location {moveOnto.Location}");
+                            throw new InsertReqOntosException($"Did not recognise location {moveOnto.ToLocation}");
                         }
 
                         locationId = toLocation.LocationId;
                     }
-                    
+
                     var insertOntosResult = await this.requisitionStoredProcedures.InsertReqOntos(
                                                 header.ReqNumber,
                                                 moveOnto.Qty,
                                                 toAdd.LineNumber,
                                                 locationId,
-                                                moveOnto.Pallet,
-                                                header.ToStockPool,
-                                                header.ToState,
-                                                "FREE");
+                                                moveOnto.ToPallet,
+                                                moveOnto.ToStockPool,
+                                                moveOnto.ToState,
+                                                "FREE"); // todo
 
                     if (!insertOntosResult.Success)
                     {
                         throw new InsertReqOntosException($"Failed in insert_req_ontos: {insertOntosResult.Message}");
                     }
                 }
             }
-            
+
+            // todo - consider what has to happen if a move is from one place to another
```

Comment:
i think it will have to call the same pick logic and if that is only doing the from side then pick it back up afterwards and add the tofields like the other pick logic does